src/send_packets.c

```diff
@@ -69,7 +69,7 @@ extern tcpedit_t *tcpedit;
 extern int debug;
 #endif
 
-static void calc_sleep_time(tcpreplay_t *ctx, struct timeval *pkt_time,
+static bool calc_sleep_time(tcpreplay_t *ctx, struct timeval *pkt_time,
```

Comment:
I added a return value to bypass the update of 'now' to reduce the number of call to gettimeofday()